src/css/index.scss

```diff
@@ -0,0 +1,103 @@
+.Info-text1{ 
+    width: 660px;
+    height: 630px;
+    background-image: repeating-linear-gradient(120deg,#3498bd,#8e44ad); 
+    color: white;
+    font-family: sans-serif;
+   }
+   
+   .h2-qu1{
+      padding: 40px 40px;
+      font-size: 40px;
+   }
+   .h2-qu2{
+      padding: 40px 40px;
+      font-size: 40px;
+      color: black;
+      font-family: sans-serif;
+      font-weight: bold;
+   }
+   
+   tr{
+       font-size: 16px;
+       line-height: 40px;   
+   }
+   .bold-font{
```

Comment:
This class naming doesn't make a good sense to me. It sounds like just make the text bold but not.